test/sqllogictest/session-window-wmr.slt

```diff
@@ -0,0 +1,406 @@
+1
+1
+1
+00:05:00
+0.01
+
+statement ok
+INSERT INTO events VALUES
+    (2, '2021-01-01 10:03:00'::timestamp, 1, '0.01');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:00:00","2021-01-01 10:08:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+2
+00:08:00
+0.02
+
+statement ok
+INSERT INTO events VALUES
+    (3, '2021-01-01 10:06:00'::timestamp, 1, '1.00');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+3
+00:11:00
+1.02
+
+statement ok
+INSERT INTO events VALUES
+    (4, '2021-01-01 10:13:00'::timestamp, 1, '0.01');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+1
+["2021-01-01 10:13:00","2021-01-01 10:18:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+3
+00:11:00
+1.02
+1
+4
+1
+00:05:00
+0.01
+
+statement ok
+INSERT INTO events VALUES
+    (5, '2021-01-01 10:15:00'::timestamp, 1, '0.01');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+1
+["2021-01-01 10:13:00","2021-01-01 10:20:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+3
+00:11:00
+1.02
+1
+4
+2
+00:07:00
+0.02
+
+statement ok
+INSERT INTO events VALUES
+    (6, '2021-01-01 10:21:00'::timestamp, 1, '0.02');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:21:00","2021-01-01 10:26:00")
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+1
+["2021-01-01 10:13:00","2021-01-01 10:20:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+3
+00:11:00
+1.02
+1
+4
+2
+00:07:00
+0.02
+1
+6
+1
+00:05:00
+0.02
+
+statement ok
+INSERT INTO events VALUES
+    (7, '2021-01-01 10:02:00'::timestamp, 2, '1.00');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:21:00","2021-01-01 10:26:00")
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+1
+["2021-01-01 10:13:00","2021-01-01 10:20:00")
+2
+["2021-01-01 10:02:00","2021-01-01 10:07:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+3
+00:11:00
+1.02
+1
+4
+2
+00:07:00
+0.02
+1
+6
+1
+00:05:00
+0.02
+2
+7
+1
+00:05:00
+1
+
+statement ok
+INSERT INTO events VALUES
+    (8, '2021-01-01 10:04:00'::timestamp, 2, '1.00');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:21:00","2021-01-01 10:26:00")
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+1
+["2021-01-01 10:13:00","2021-01-01 10:20:00")
+2
+["2021-01-01 10:02:00","2021-01-01 10:09:00")
+
+query TTTTT
+SELECT * FROM user_session_worth
+----
+1
+1
+3
+00:11:00
+1.02
+1
+4
+2
+00:07:00
+0.02
+1
+6
+1
+00:05:00
+0.02
+2
+7
+2
+00:07:00
+2
+
+statement ok
+INSERT INTO events VALUES
+	(9, '2021-01-01 10:17:00'::timestamp, 1, '0.01');
+
+query TT
+SELECT * FROM event_session
+----
+1
+["2021-01-01 10:00:00","2021-01-01 10:11:00")
+1
+["2021-01-01 10:13:00","2021-01-01 10:26:00")
+2
+["2021-01-01 10:02:00","2021-01-01 10:09:00")
+
+query TTTTT
```

Comment:
Is there a way to render each row of the expected result in a single line?

Reply:
slt doesn't support this but we can fake it in SQL